library/core/class.controller.php

```diff
@@ -287,6 +287,10 @@ public function __construct() {
 
         parent::__construct();
         $this->ControllerName = strtolower($this->ClassName);
+
+        // Add current theme path to GDN object
+        $themeInfo = Gdn::themeManager()->getThemeInfo(Gdn::themeManager()->currentTheme());
```

Comment:
The theme manager is deprecated.

The easiest way to get this now is likely with:

```php
/** Addon $currentTheme */
$currentTheme = Gdn::getContainer()->get(ContainerUtils::currentTheme());
```